testgrid/config/config.yaml

```diff
@@ -815,8 +815,8 @@ test_groups:
 # Canonical Distribution of Kubernetes (contact: @chuckbutler on github)
 - name: canonical-kubernetes-e2e-gce
   gcs_prefix: canonical-kubernetes-tests/logs/kubernetes-gce-e2e-node
-# Ubuntu GKE (contact: @vorlonofportland on github)
-- name: ubuntu-gke
+# Ubuntu-1604-v20170420 node e2e tests on GCE (contact: @vorlonofportland on github)
```

Comment:
I created a @kubernetes/ubuntu-image team on github. Maybe contact that team instead of an individual?